cpp/benchmarks/common/generate_input.cu

```diff
@@ -33,6 +33,7 @@
 #include <rmm/device_uvector.hpp>
 
 #include <thrust/binary_search.h>
+#include <thrust/copy.h>
```

Comment:
Note to reviewers: all the `#include` changes were automatically generated by a script I wrote. See notes on #10489.